app/enrichment/enrich-header.js

```diff
@@ -4,16 +4,18 @@ const createInvoiceNumber = require('./create-invoice-number')
 const getFrn = require('./get-frn')
 const { AP } = require('../ledgers')
 const { convertToDaxDate } = require('../date-convert')
+const GBP = 'GBP'
 
 const enrichHeader = async (paymentRequest, scheme) => {
   paymentRequest.correlationId = paymentRequest.correlationId ?? uuidv4()
   paymentRequest.invoiceNumber = createInvoiceNumber(paymentRequest)
-  paymentRequest.schemeId = scheme.schemeId
+  paymentRequest.schemeId = scheme?.schemeId
```

Comment:
Does this smell as to an ineffective use of Joi schema validation with other objects in our application outside of those entering?